packages/form-participation/src/validators.ts

```diff
@@ -0,0 +1,75 @@
+import { Validator } from './index';
+
+export interface RequiredHost extends HTMLElement {
+  required: boolean;
+};
+
```

Comment:
general question: how many default validators need to be provided? if there are more, should we add them here, or as a fast follow PR? I don't mind PRing some more default validators to add to this lib

Reply:
As above, happy to have that discussion. I didn't want to spend too much time on it just yet, but leave that open for discussion. Would we need min, max, isNumber, isDate, etc?

Reply:
I think following up is a good call. Also, ensuring that documentation clearly outlines making your own so that people can test/vend/leverage their own over time is good.

Reply:
OK, coming up with a list of default validators:

- [ ] Required/valueMissing
- [ ] minlength/tooShort
- [ ] maxlength/tooLong
- [ ] isNumber/typeMismatch
- [ ] isDate/typeMismatch
- [ ] min/rangeUnderflow (number and date?)
- [ ] max/rangeOverflow (number and date?)
- [ ] pattern (regex)/patternMismatch

Any others we can think of?